src/main/java/tech/pegasys/pantheon/plugin/services/MetricsSystem.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.plugin.services;
+
+import tech.pegasys.pantheon.plugin.services.metrics.Counter;
+import tech.pegasys.pantheon.plugin.services.metrics.LabelledMetric;
+import tech.pegasys.pantheon.plugin.services.metrics.MetricCategory;
+import tech.pegasys.pantheon.plugin.services.metrics.OperationTimer;
+
+import java.util.function.DoubleSupplier;
+import java.util.function.IntSupplier;
+import java.util.function.LongSupplier;
+
+/** An interface for creating various Metrics components and retrieving observations. */
+public interface MetricsSystem {
+
+  /**
+   * Creates a Counter.
+   *
+   * @param category The {@link MetricCategory} this counter is assigned to.
+   * @param name A name for this metric.
+   * @param help A String containing helpful information.
+   * @return The created Counter instance.
+   */
+  default Counter createCounter(
+      final MetricCategory category, final String name, final String help) {
+    return createLabelledCounter(category, name, help, new String[0]).labels();
+  }
+
+  /**
+   * Creates a Counter with assigned labels.
+   *
+   * @param category The {@link MetricCategory} this counter is assigned to.
+   * @param name A name for this metric.
+   * @param help A String containing helpful information.
+   * @param labelNames An array of labels to assign to the Counter.
+   * @return The created LabelledMetric instance.
+   */
+  LabelledMetric<Counter> createLabelledCounter(
+      MetricCategory category, String name, String help, String... labelNames);
+
+  /**
+   * Creates a Timer.
+   *
+   * @param category The {@link MetricCategory} this timer is assigned to.
+   * @param name A name for this metric.
+   * @param help A String containing helpful information.
```

Comment:
Can we change all these `help` descriptions to:
```suggestion
   * @param help A human readable description of the metric.
```